### ISSUE TITLE
不需要加softmax么？

### ISSUE THREAD

**Issue by zhh9211** [MorvanZhou/PyTorch-Tutorial#65]
https://github.com/MorvanZhou/PyTorch-Tutorial/blob/51c6c660c7658c13c2db9107a1cc54dadf240bd4/tutorial-contents/402_RNN_classifier.py#L74

**Comment by MorvanZhou** [comment]
在pytorch的定义中，是使用logits而不是softmax后的结果。你可以参考一下文档。https://pytorch.org/docs/stable/nn.html#crossentropyloss

loss_func = nn.CrossEntropyLoss()                       # the target label is not one-hotted
